ex1-shai/checkNcdu.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#Install ncdu
+which ncdu > /dev/null 2>&1
+if [[ $? -eq 1 ]]; then	
+    echo "Ncdu is not currently installed on Linux, are you interested in installing it? select the option that you prefer[Y/N]"
+    read answer
+    if [[ $answer = "Y" ]]; then
+        echo "Installing ncdu"
+        yum install -y epel-release
+        sudo yum provides ncdu 
```

Comment:
What this command used for?

Reply:
To show package that contains ncdu.

Reply:
why is it needed?

Reply:
I thought that it would be more convenient to see the package.
You're right don't need it. Delete the line?

Reply:
convenient for who?

Reply:
For those who install,
Do you think this is unnecessary?